ChangeLog

```diff
@@ -1,5 +1,11 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+Bugfix
+   * Fix handshake failure in NIST suite b, where the key was determined as
+     MBEDTLS_ECKEY instead of MBEDTLS_ECDSA.
```

Comment:
I don't believe this is the best way to describe the issue to a user: ChangeLog entries should be about observable behaviour, not internal details. Also "suite b" is vague, when first reading this I thought it was about `configs/config-suite-b.h` - and the issue is not just inside SSL but also for any certificate verification. I suggest something like:
```
Fix mbedtls_x509_crt_profile_suiteb, which used to reject all certificates with flag MBEDTLS_X509_BADCERT_BAD_PK even when the key type was correct.  In the context of SSL, this resulted in handshake failure.
```
We probably want to credit the person who reported the bug too.

Reply:
I thought about crediting the person, but I don't have his last name